man/systemd-system.conf.xml

```diff
@@ -171,6 +171,126 @@
         effect if a hardware watchdog is not available.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>IOCostQOSPath=</varname></term>
+        <listitem><para> Configure the absolute path to device to be configured
+        in io.cost.qos file of the root cgroup.</para></listitem>
+      </varlistentry>
```

Comment:
Hmm, why does this take *one* path? i.e. shouldn't it be possible to configure the cost for multiple devices?

Why isn't this modelled afte IODeviceLatencyTargetSec=, IODeviceWeight=?